binanceinfo.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+from binance.client import Client
+import os
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
+
+API_KEY = (os.getenv('API_KEY'))
+API_SECRET = (os.getenv('API_SECRET'))
+
+client = Client(API_KEY, API_SECRET)
+
+
+# print(client.get_account())
+info = client.get_exchange_info()
+print(info.keys())
+symbols = info["symbols"]
+
+print(symbols[1])
+
```

Comment:
You need warp up this block to generator construction to use it in future after import to `main.py`